e2e/testdata/fn-render/resource-has-pkgname-prefix/mysql/Kptfile

```diff
@@ -0,0 +1,9 @@
+apiVersion: kpt.dev/v1alpha1
```

Comment:
We need a way to test packages in top-level `package-examples/` directory. They're used throughout the documentation. That also avoids duplicating it again in `e2e/testdata`.

The only reason for not having .expected in those packages is if it gets fetched the user's local filesystem and confuses them. WDYT?

Reply:
Yes, testing `package-examples` is a MUST, we can save our users and us from troubles. You are right, we can't have .expected in those packages, it's going to be super confusing for users. One way we can solve this is:

- Have a `package-examples-expected` directory that contains the `.expected` for each example package.
- We write `TestPkgExamples` method in our e2e tests that copies the `.expected` for each test from `package-examples-expected` while staging the test and then drive the test using regular flow.

WDYT ?

Reply:
I like that idea.

Reply:
Filed https://github.com/GoogleContainerTools/kpt/issues/1697 to track this.